core/src/test/scala/org/apache/spark/deploy/SparkHadoopUtilSuite.scala

```diff
@@ -93,4 +139,41 @@ class SparkHadoopUtilSuite extends SparkFunSuite {
     assert(hadoopConf.get(key) === expected,
       s"Mismatch in expected value of $key")
   }
+
+  /**
+   * Assert that a hadoop configuration option has the expected value
+   * and has the expected source.
+   *
+   * @param hadoopConf configuration to query
+   * @param key        key to look up
+   * @param expected   expected value.
+   * @param expectedSource string required to be in the property source string
+   */
+  private def assertConfigMatches(
+    hadoopConf: Configuration,
+    key: String,
+    expected: String,
+    expectedSource: String): Unit = {
```

Comment:
Indentation: Two more spaces.
```scala
  private def assertConfigMatches(
       hadoopConf: Configuration,
       key: String,
       expected: String,
       expectedSource: String): Unit = {
```

Reply:
will need to tweak my scala ide rules, i see